docs/dev/developer-guides/contracts/system-contracts.md

```diff
@@ -1,7 +1,8 @@
 # Understanding system contracts
-To keep the zero-knowledge circuits as simple as possible and enable simple extensions, a large chunk of the logic of zkSync was moved to the so-called "system contracts" -- a set of contracts that have special privileges and serve special purposes, e.g. deployment of contracts, making sure that the user pays only once for publishing contracts' calldata, etc.
+To keep the zero-knowledge circuits as simple as possible and enable simple extensions. A large chunk of the logic of zkSync was moved to the so-called "system contracts" – a 
```

Comment:
this should not be two separate phrases.

Reply:
what is "this"?

Reply:
if you mean the line break, it is done automatically by the ide. It displays as one sentence on html

Reply:
I mean, "and enable simple extensions. A large chunk of the logic" should be "and enable simple extensions, a large chunk of the logic".